.github/workflows/continuous-integration-workflow.yml

```diff
@@ -15,6 +27,14 @@ jobs:
         name: Check shell scripts
         run: |
           shellcheck sonarqube/*.sh
+      -
+        name: Build image
+        working-directory: sonarqube
+        run: |
+          docker build \
+          --build-arg sonarVersion=${{ matrix.version }} \
+          --build-arg sonarDistributionUrl="https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-${{ matrix.version }}.zip" \
+          -t sonarcube .
```

Comment:
`test.sh` already builds the image, so this should not be necessary?

Reply:
I remove the docker build for sonarcube